v2/apiserver/internal/core/projects.go

```diff
@@ -200,6 +206,71 @@ func (p *projectsService) Create(
 			errors.Wrapf(err, "error storing new project %q", project.ID)
 	}
 
+	// Make the current user an admin, developer, and user of the project
```

Comment:
If we _don't_ do this, not only can no one update or use this project, but the project won't even have a user with sufficient permissions to change that. i.e. If we don't do this, every project that is created is completely useless unless you are the root user.